ros2_test_framework/include/ros2_test_framework/publisher_mock.hpp

```diff
@@ -79,6 +79,13 @@ class PublisherMock : public MockBase
    */
   MOCK_METHOD(void, publish, (const MessageT & msg));
 
+  /**
+   * @brief Set the fake subscriptions count.
+   *
+   * @param count number of fake subscriptions
+   */
+  void set_subscription_count(size_t count) { subscriptions_count_ = count; }
```

Comment:
Shouldn't it be `setSubscriptionCount`?
```suggestion
  void setSubscriptionCount(size_t count) { subscriptions_count_ = count; }
```

Reply:
ok renamed